css/main.sass

```diff
@@ -351,10 +350,10 @@ main > img, .img-row, table
         height: 100%
 
 table img
-    width: 100%
+        width: 100%
 
 main h1
-    text-align: center
+        text-align: center
```

Comment:
Not sure why all the whitespace changes but not really a big deal

Reply:
i had done alt + shift + f in habit